test/pytest/test_qonnx.py

```diff
@@ -309,6 +336,79 @@ def test_sep_conv(sep_conv_model, backend):
     np.testing.assert_allclose(y_qonnx.ravel(), y_hls4ml.ravel(), atol=1e-2, rtol=1)
 
 
+class EmptyFilledRoI(Transformation):
```

Comment:
Why is this in the test? Why do we need to remove a filled ROI? Can't we ignore it or assert that it must not be set?

Reply:
The RoI must be filled to run inferences in QONNX (see here: https://github.com/fastmachinelearning/qonnx/issues/150) so it has to be set while using QONNX model for inferences. If it is left filled during hls4ml ingestion it is considered as an input since in QONNX graph RoI is a node, generating garbage code that breaks the hls code.